upgrade/prepare

```diff
@@ -18,8 +18,11 @@
 UPDATE_DIR=${UPDATE_DIR:-/var/dlpx-update}
 
 function die() {
+	exit_code=$1
+	[[ -z "${exit_code//[0-9]/}" ]] && [[ -n "$exit_code" ]] || exit_code=1
```

Comment:
What you probably want is something like:
```
function die() {
        exit_code=$1
        # Use first argument as exit code, if provided.
        if [[ -z "${exit_code//[0-9]/}" ]]; then
                shift
        else
                exit_code=1
        fi
        echo "$(basename "$0"): $*" >&2
        exit ${exit_code}
}
```
This way if there is no number provided as first argument, it will print the whole string and exit with `1`

Reply:
Done.